crates/cdk/src/test.rs

```diff
@@ -15,7 +15,7 @@ pub struct TestCmd {
     package: PackageCmd,
 
     /// Path to configuration file in YAML format
-    #[clap(short, long, value_name = "PATH")]
+    #[clap(short, long, value_name = "PATH", default_value = "sample-param.yaml")]
```

Comment:
`param` => `params` to be consistent with the label used in `Connector.toml`

Reply:
rename this to `sample-config.yaml`.  This PR needs to be merged after `generate` otherwise there will be no default